src/EnergyPlus/PhotovoltaicThermalCollectors.cc

```diff
@@ -103,11 +107,12 @@ namespace PhotovoltaicThermalCollectors {
     //  the first model is a "simple" or "ideal" model useful for sizing, early design, or policy analyses
     //  Simple PV/T model just converts incoming solar to electricity and temperature rise of a working fluid.
 
-    int constexpr SimplePVTmodel(1001);
-
     Real64 constexpr SimplePVTWaterSizeFactor(1.905e-5); // [ m3/s/m2 ] average of collectors in SolarCollectors.idf
 
+    int NumPVT(0); // count of all types of PVT in input file
+
     PlantComponent *PVTCollectorStruct::factory(EnergyPlusData &state, std::string_view objectName)
+
```

Comment:
I missed this entire file, so there will be another set of comments incoming. Sorry.